samples/recognize.js

```diff
@@ -197,6 +197,9 @@ async function asyncRecognize(
     sampleRateHertz: sampleRateHertz,
     languageCode: languageCode,
   };
+
+  // Note that transcription is limited to 60 seconds audio. 
```

Comment:
👋  looks like you need to run `npm run fix` here

Reply:
It's fixed now.

Reply:
@googlebot I signed it!